### ISSUE TITLE
ol.source.OSM tiles do not clear after zoom/pan event

### ISSUE THREAD

**Issue by dbauszus** [openlayers/ol3#4806]
I am using a series of OSM style tile layers in various maps. So far I have been working with ol 3.11.1 where this problem does not exist. I upgraded today to 3.13.1 where tiles are not always cleared after a pan/zoom event.

Have there been changes to the ol.source.OSM or ol.layer.Tile implementation between 3.11 and 3.13?

Cheers,
Dennis

Edit: I test the behaviour on Edge, Firefox, Chrome

![capture](https://cloud.githubusercontent.com/assets/6732788/12925136/bd8b16fe-cf54-11e5-91cb-6095cc030385.PNG)

```
var layerOSM_lightLabels = new ol.layer.Tile({
    source: new ol.source.OSM({
        url: 'https://cartodb-basemaps-{a-d}.global.ssl.fastly.net/light_only_labels/{z}/{x}/{y}.png'
    })
});
```

**Comment by fredj** [comment]
Try to add the `opaque` option:
```js
var layerOSM_lightLabels = new ol.layer.Tile({
    source: new ol.source.OSM({
        url: 'https://cartodb-basemaps-{a-d}.global.ssl.fastly.net/light_only_labels/{z}/{x}/{y}.png',
        opaque: false
    })
});
```

**Comment by dbauszus** [comment]
No change I am afraid. The label tiles are still not cleared.

![capture](https://cloud.githubusercontent.com/assets/6732788/12926178/18645b9e-cf5a-11e5-8ed9-d89df69eaffa.PNG)

**Comment by petrsloup** [comment]
@fredj The `opaque` option to `ol.source.OSM` was added in https://github.com/openlayers/ol3/pull/4769, but it's not part of v3.13.1, right?

**Comment by fredj** [comment]
@petrsloup you're right: this fix is not in v3.13.1.
@dbauszus can you try with [master](http://openlayers.org/en/master/build/ol.js) ?

**Comment by dbauszus** [comment]
This is fixed in the master. The tile layer with the labels clears correctly with the opaque: false setting.
